linux-documentation/secure_launch_overview.rst

```diff
@@ -0,0 +1,110 @@
+
+- Secure Launch Overview (secure_launch_overview.rst)
+  - TrenchBoot Project
+  - Secure Launch Goals
+  - Functional Overview
+
+
+----
+__From last Patch Set:__
+
+
+=============
+Secure Launch
+=============
+
+Overview
+========
+
+Prior to the start of the TrenchBoot project, the only active Open
+Source project supporting dynamic launch was Intel's tboot project to
+support their implementation of dynamic launch known as Intel Trusted
+eXecution Technology (TXT). The approach taken by tboot was to provide
+an exokernel that could handle the launch protocol implemented by
+Intel's special loader, the SINIT Authenticated Code Module (ACM [3]_)
+and remained in memory to manage the SMX CPU mode that a dynamic launch
+would put a system. While it is not precluded from being used for doing
+a late launch, tboot's primary use case was to be used as an early
+launch solution. As a result the TrenchBoot project started the
+development of Secure Launch kernel feature to provide a more
+generalized approach. The focus of the effort is twofold, the first is to
+make the Linux kernel directly aware of the launch protocol used by
+Intel as well as AMD, and the second is to make the Linux kernel be able
```

Comment:
and other vendors?

Reply:
yah yah, i know. this was written very x86 specific (though hygon is supposedly looking to support DRTM on their chips).

Reply:
Very interesting, I assume this maybe driven by Microsoft Secured Core PC for servers.